csharp/src/Ice/IPEndpoint.cs

```diff
@@ -29,16 +34,21 @@ internal abstract class IPEndpoint : Endpoint
 
         internal const ushort DefaultIPPort = 4062;
 
+        /// <summary>Whether IPv6 sockets created from this endpoint are dual-mode or IPv6 only.</summary>
```

Comment:
Indicates ...?
Should be `IsIPv6Only` like other bool properties.

Reply:
If that's what we usually do, sure.